calls/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import include, path
+from rest_framework import routers
+from .views import CallRecordViewSet, BillViewSet
+
+
+router = routers.DefaultRouter()
+router.register(r'Call-Record', CallRecordViewSet, basename="CallRecord")
```

Comment:
Sugiro sempre usar nome de recurso em minúsculo.

Motivos:
- Mais comum ser tudo minisculo;
- Tem algumas ferramenta que podem converter tudo pra minusculo e sua API não seria acessada;

Podem existir casos e empresa que precise ser como colocou aqui, dai vai de empresa pra empresa.

Referencia:
https://restfulapi.net/resource-naming/ - no topico 5 - Use lowercase letters in URIs